utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -162,6 +165,8 @@ impl ParamSetter {
     }
 }
 
+pub type SysCallInvokeInstructions = Vec<Instruction>;
```

Comment:
Strange alias

Reply:
It's returned from fn as `Vec<SysCallInvokeInstructions>`, so without it return type will be like `Vec<Vec<_>>`